website/pages/docs/configuration/client.mdx

```diff
@@ -372,6 +378,35 @@ client {
 - `read_only` `(bool: false)` - Specifies whether the volume should only ever be
   allowed to be mounted `read_only`, or if it should be writeable.
 
+### `host_network` Stanza
+
+The `host_network` stanza is used to register additional host networks with
+the node that can be used when port mapping.
+
+The key of the stanza corresponds to the name of the network used in the
+[`host_network`](/docs/job-specification/network#host-network).
```

Comment:
Sorry for the late review, but the link here is broken. It needs to be `/docs/job-specification/network#host-networks` (with an `s` at the end)

Reply:
Good catch I’ll fix it in my next round of docs